src/ansys/geometry/core/modeler.py

```diff
@@ -318,13 +318,13 @@ def run_discovery_script_file(
         ----------
         file_path : str
             Path of the file. The extension of the file must be included.
-        script_args : dict[str, str]
-            Arguments to pass to the script.
-        import_design : bool, default: False
+        script_args : dict[str, str], optional.
+            Arguments to pass to the script. By default, `None`
```

Comment:
```suggestion
            Arguments to pass to the script. By default, ``None``
```
Align with library standards... we are using double ticks instead of single ticks...